### ISSUE TITLE
[Programmatic access - ADF - Azure data factory] Tooltip contains special characters for Azure data factory control at left panel.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/AzureStorageExplorer#3849]
**User Experience:**
Screen reader user will be misguide if programmatically defined doesn't appear visually as user won't get the actual context of control.

**Test Environment:**
Application Version: 1.16.0-dev
Build Number: 20200921.1

**Repro Steps:​**
- Launch Azure Storage Explorer app.
- Navigate to visual control 'Add data factory control' at left panel.
- Verify tooltip of visual control 'Add data factory control'.

**Actual Result:**
Tooltip contains special characters for Azure data factory control at left panel. Tooltip is defined as 'Azure_data_factory' for visual control 'Add data factory control'. 

**Expected Result:**
Tooltip must not contain special characters for Azure data factory control at left panel. Tooltip should be defined as 'Azure data factory' for visual control 'Add data factory control'. 

[MAS2.5.3_Azure data factory_Tooltip.zip](https://github.com/microsoft/AzureStorageExplorer/files/5641691/MAS2.5.3_Azure.data.factory_Tooltip.zip)
